data/sql/updates/pending_db_world/rev_1728239921994699100.sql

```diff
@@ -0,0 +1,20 @@
+UPDATE `creature_template` SET `unit_flags` = 33587202 WHERE (`entry` = 28782);
+
+INSERT INTO creature_text
+(CreatureID, GroupID, ID, `Text`, `Type`, `Language`, Probability, Emote, Duration, Sound, BroadcastTextId, TextRange, comment)
+VALUES(28782, 0, 0, '%s rears up, beckoning you to ride it.', 16, 0, 100.0, 0, 0, 0, 29069, 0, 'Archerus Deathcharger');
```

Comment:
Before an insert you have to put a DELETE, Moreover, please add the backtick in all columns and table name

Reply:
I've implemented the changes mentioned here.